firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt

```diff
@@ -111,12 +105,12 @@ class FilterDialogFragment : DialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        rootView = inflater.inflate(R.layout.dialog_filters, container, false)
+        binding = DialogFiltersBinding.inflate(inflater, container, false)
 
-        rootView.buttonSearch.setOnClickListener { onSearchClicked() }
-        rootView.buttonCancel.setOnClickListener { onCancelClicked() }
+        binding.buttonSearch.setOnClickListener { onSearchClicked() }
+        binding.buttonCancel.setOnClickListener { onCancelClicked() }
 
-        return rootView
+        return binding.root
```

Comment:
As per my comment above, this would become:
```kotlin
_binding = DialogFiltersBinding.inflate(inflater, container, false)
// click listeners
return binding.root
```